src/main/steps/applicant/yourdocuments/alldocuments/applicationmade/content.test.ts

```diff
@@ -0,0 +1,54 @@
+import languageAssertions from '../../../../../../test/unit/utils/languageAssertions';
```

Comment:
For both applicant and respondent 'your document' section, please follow the same naming conventions for file and directory names as used for other section.